README.md

```diff
@@ -4,26 +4,70 @@
+Key                     | Purpose                                             | Default Value
+----------------------- | --------------------------------------------------- | --------------
+DBNAME                  | postgres database name                              | dojot_devm
+DBHOST                  | postgres database host                              | postgres
+DBUSER                  | postgres database user                              | postgres
+DBPASS                  | postgres database password                          | none
+DBDRIVER                | postgres database driver                            | postgresql+psycopg2
+CREATE_DB               | option to create the database                       | True
+KAFKA_HOST              | kafka host                                          | kafka
+KAFKA_PORT              | kafka port                                          | 9092
+STATUS_TIMEOUT          | kafka timeout                                       | 5
+BROKER                  | kafka topic subject manager                         | http://data-broker
+REDIS_HOST              | redis host                                          | device-manager-redis
+REDIS_PORT              | redis port                                          | 6379
+SUBJECT                 | subject of device-manager                           | dojot.device-manager.device
+DEV_MNGR_CRYPTO_PASS    | password of crypto                                  | none
+DEV_MNGR_CRYPTO_IV      | inicialization vector of crypto                     | none
+DEV_MNGR_CRYPTO_SALT    | salt of crypto                                      | none
+
+## Internal Messages
+
+There are some messages that are published by DeviceManager through Kafka. These messages are notifications of device management operations, and they can be consumed by any component interested in them, such as IoT agents.
+
+Event                   | Service                                             | Message Type
+----------------------- | --------------------------------------------------- | --------------
+Device creation         | dojot.device-manager.device                         | Creation message
+Device update           | dojot.device-manager.device                         | Update message
+Device removal          | dojot.device-manager.device                         | Removal message
+Device actuation        | dojot.device-manager.device                         | Actuation message
+Template update         | dojot.device-manager.device                         | Template update message
 
 ## How to run
 
+A docker image is available on dockerhub for pull [here](https://hub.docker.com/r/dojot/device-manager)
+
 If you really need to run DeviceManager as a standalone process (without dojot's wonderful [docker-compose](https://github.com/dojot/docker-compose)), we suggest using the minimal compose
 file available under `docker/compose.yml`. That contains only the set of external systems (postgres
```

Comment:
I think the correct file name is docker-compose.yml

Reply:
The file actually is called compose.yml, but I missed the directory where the file is (local/compose.yml).
Maybe rename the compose.yml to docker-compose.yml?